docs/team/zhou-jiahao-1998.md

```diff
@@ -3,6 +3,46 @@ layout: page
 title: Zhou Jiahao's Project Portfolio Page
 ---
 
-### Project: Spam
+### Project: SPAM
+
+SPAM is a desktop application that is made for fast-typing marketers to handle large quantities of marketing leads and
+contacts efficiently and precisely. It is a CLI application that has UNIX-like command line syntax, and sports an
+elegant and modern GUI created with JavaFX. The application is written in Java, and has about 15 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Encryption integration
+    * What it does: allows SPAM to read and write from the encrypted file.
+    * Justification: This feature makes sure that the data file stays encrypted at all times.
+    * Credits: Pair programming with @huizhuansam.
+
+
+* **New Feature**: Password protection
+    * What it does: Adds a password to the application.
+    * Justification: This feature improves the security of the application data and works hand in hand with the encryption feature.
+    * Highlights: One time set up on new data. Able to share the encrypted data file across devices using the same password. The password is not stored anywhere within the application.
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=13&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=Zhou-Jiahao-1998&tabRepo=AY2122S1-CS2103T-W13-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+
+
+* **Project management**:
+    * Managed releases `v1.3.trial` on GitHub
+  
+
+* **Enhancements to existing features**:
+    * Added support to search by `tags` in `find`.
+
+
+* **Documentation**:
+    * User Guide:
+        * Added password set up documentation in `Getting Started`.
+        * Added documentation for the features `find` and `password`.
+    * Developer Guide:
+        * Added implementation details of the `Encryption` and `Password` feature.
+
+
+* **Community**:
+    * PRs reviewed (with non-trivial review comments): [\#91](), [\#104](), [\#116]()
+    * Reported bugs and suggestions for other teams in the class
```

Comment:
Do add links to evidence. Your PED issues can be found in your personal Github issues tab.